torch/_prims/__init__.py

```diff
@@ -157,6 +159,30 @@ class RETURN_TYPE(Enum):
     INPLACE = (2,)
 
 
+def _wrap_tensor_meta(f):
+    def wrap(t):
+        if isinstance(t, torch.Tensor):
+            return TensorMeta(t)
+        else:
+            return t
+
+    def unwrap(t):
+        # TODO: doesn't setup aliasing relation on views correctly
```

Comment:
worth adding assert here ?

Reply:
I'll think about it. Not sure exactly how the assert should go.